client/src/partials/home/units/debtors.js

```diff
@@ -0,0 +1,35 @@
+angular.module('bhima.controllers')
+.controller('DashboardDebtorController', DashboardDebtorController);
+
+DashboardDebtorController.$inject = ['DashboardService'];
+
+function DashboardDebtorController(Dashboard) {
+  var vm = this;
+
+  // @TODO move to service to deal with assigning application colours
+  vm.darkBlueGradient = ['#085484', '#3a6893', '#597ba2', '#7490b1', '#90a5c0', '#abbbd0',
+    '#c7d1df', '#e3e8ef', '#ffffff'];
+
+  vm.owedGraph = {
+    colors : vm.darkBlueGradient,
+    options : {
+      legend : {
+        display : true
+      }
+    }
+  };
+
+  Dashboard.debtors()
+    .then(function (result) {
+      var debtorGroups = result.data.debtorGroups;
+      vm.reportCached = result.timeCached;
+
+      vm.owedGraph.data = debtorGroups.map(function (group) {
```

Comment:
I feel obliged to comment that you could have done this in one loop with a reduce.  But this is fine :+1: